extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -373,6 +371,16 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     return this.uiHandler.createPlaceholder();
   }
 
+  /**
+   * @returns {!Promise<?CONSENT_POLICY_STATE>}
+   */
+  getConsentState() {
+    const consentPolicyId = super.getConsentPolicy();
```

Comment:
I can't remember does this change? Trying to think of a clever way to not repeat most of this method above.